From c33cd8897b3c05ad53f61288fca781e25a1f56d3 Mon Sep 17 00:00:00 2001
From: Valerii Chubar <valerii_chubar@epam.com>
Date: Fri, 4 Jun 2021 22:55:12 +0300
Subject: [PATCH] copyscript: Set GFX Library List to empty string

We don't need to populate meta-renesas with native gfx software
packages. All what we need from the copyscript is to copy native
multimedia software packages. In order to not re-pack package archives
(to remove corresponding packages), set _gfx_list to empty string
to just skip copying them.

Signed-off-by: Oleksandr Tyshchenko <oleksandr_tyshchenko@epam.com>
---
 .../copyscript/copy_proprietary_softwares.sh     | 16 ++++++++--------
 1 file changed, 8 insertions(+), 8 deletions(-)

diff --git a/meta-rcar-gen3/docs/sample/copyscript/copy_proprietary_softwares.sh b/meta-rcar-gen3/docs/sample/copyscript/copy_proprietary_softwares.sh
index 798bbc9..d6fe45f 100755
--- a/meta-rcar-gen3/docs/sample/copyscript/copy_proprietary_softwares.sh
+++ b/meta-rcar-gen3/docs/sample/copyscript/copy_proprietary_softwares.sh
@@ -129,14 +129,14 @@ _adsp_km_list="adsp_driver,RTM8RC0000ZNA2DS00JFL3E,RTM8RC0000ZNA2DS00JFL3E.tar.g
 # gfx_list="<package_name(user)>,<copy_file_name(user)>,<package_name(kernel)>,<copy_file_name(kernel)> \
 #           <package_name(user)>,<copy_file_name(user)>,<package_name(kernel)>,<copy_file_name(kernel)> \
 #           <package_name(user)>,<copy_file_name(user)>,<package_name(kernel)>,<copy_file_name(kernel)>"
-_gfx_list="RTM8RC7795ZG300Q10JPL3E,r8a77951_linux_gsx_binaries_gles.tar.bz2,RCH3G001L5101ZDO,GSX_KM_H3.tar.bz2 \
-           RTM8RC7796ZG300Q10JPL3E,r8a77960_linux_gsx_binaries_gles.tar.bz2,RCM3G001L5101ZDO,GSX_KM_M3.tar.bz2 \
-           RTM8RC7796ZG300Q50JPL3E,r8a77965_linux_gsx_binaries_gles.tar.bz2,RCN3G001L5101ZDO,GSX_KM_M3N.tar.bz2 \
-           RTM8RC7799ZG300Q00JPL3E,r8a77990_linux_gsx_binaries_gles.tar.bz2,RCE3G001L5101ZDO,GSX_KM_E3.tar.bz2 \
-           INFRTM8RC7795ZG300Q10JPL3E,INF_r8a77951_linux_gsx_binaries_gles.tar.bz2,RCH3G001L5101ZDO,GSX_KM_H3.tar.bz2 \
-           INFRTM8RC7796ZG300Q10JPL3E,INF_r8a77960_linux_gsx_binaries_gles.tar.bz2,RCM3G001L5101ZDO,GSX_KM_M3.tar.bz2 \
-           INFRTM8RC7796ZG300Q50JPL3E,INF_r8a77965_linux_gsx_binaries_gles.tar.bz2,RCN3G001L5101ZDO,GSX_KM_M3N.tar.bz2 \
-           INFRTM8RC7799ZG300Q00JPL3E,INF_r8a77990_linux_gsx_binaries_gles.tar.bz2,RCE3G001L5101ZDO,GSX_KM_E3.tar.bz2"
+#_gfx_list="RTM8RC7795ZG300Q10JPL3E,r8a77951_linux_gsx_binaries_gles.tar.bz2,RCH3G001L5101ZDO,GSX_KM_H3.tar.bz2 \
+#           RTM8RC7796ZG300Q10JPL3E,r8a77960_linux_gsx_binaries_gles.tar.bz2,RCM3G001L5101ZDO,GSX_KM_M3.tar.bz2 \
+#           RTM8RC7796ZG300Q50JPL3E,r8a77965_linux_gsx_binaries_gles.tar.bz2,RCN3G001L5101ZDO,GSX_KM_M3N.tar.bz2 \
+#           RTM8RC7799ZG300Q00JPL3E,r8a77990_linux_gsx_binaries_gles.tar.bz2,RCE3G001L5101ZDO,GSX_KM_E3.tar.bz2 \
+#           INFRTM8RC7795ZG300Q10JPL3E,INF_r8a77951_linux_gsx_binaries_gles.tar.bz2,RCH3G001L5101ZDO,GSX_KM_H3.tar.bz2 \
+#           INFRTM8RC7796ZG300Q10JPL3E,INF_r8a77960_linux_gsx_binaries_gles.tar.bz2,RCM3G001L5101ZDO,GSX_KM_M3.tar.bz2 \
+#           INFRTM8RC7796ZG300Q50JPL3E,INF_r8a77965_linux_gsx_binaries_gles.tar.bz2,RCN3G001L5101ZDO,GSX_KM_M3N.tar.bz2 \
+#           INFRTM8RC7799ZG300Q00JPL3E,INF_r8a77990_linux_gsx_binaries_gles.tar.bz2,RCE3G001L5101ZDO,GSX_KM_E3.tar.bz2"
 
 # ICCOM
 # Please add ICCOM to "_iccom_list"
-- 
2.17.1

